toolkit/primitives/domain/src/lib.rs

```diff
@@ -300,9 +300,12 @@ impl MainchainKeyHash {
 	}
 }
 
+/// ECDSA signature of MainchainPrivateKey, 64 bytes.
+const MAINCHAIN_SIGNATURE_LEN: usize = MAINCHAIN_PRIVATE_KEY_LEN * 2;
```

Comment:
Is there any relation between private key length and signature length?
Why not use literal 64 ?
Why not use 2*MAINCHAIN_PUBLIC_KEY_LEN ?